src/apps/saddle/ethereum/saddle.pool.definitions.ts

```diff
@@ -36,6 +36,11 @@ export const SADDLE_BASEPOOL_DEFINITIONS: CurvePoolDefinition[] = [
     swapAddress: '0x101cd330d088634b6f64c2eb4276e63bf1bbfde3',
     tokenAddress: '0x1b4ab394327fdf9524632ddf2f0f04f9fa1fe2ec',
   },
+  // Saddle Frax 3Pool
```

Comment:
This is on Ethereum Mainnet - the Farm where the $SDL incentives are coming from is already live with the Zapper/Saddle integration.